src/transaction.ts

```diff
@@ -653,7 +653,7 @@ export class TransactionComputer {
 
     const serialized = JSON.stringify(plainTransaction);
 
-    return new Uint8Array(Buffer.from(serialized));
+    return Buffer.from(serialized);
```

Comment:
Well, in theory it would break compatibility, but this has not been yet released so there can't be any apps that use it. And since `Buffer` implements `Uint8Array` I assume it's all right.

Reply:
Indeed, `TransactionComputer` is new in v13.